pkg/option/config.go

```diff
@@ -881,6 +881,9 @@ const (
 	// XDPModeNative for loading progs with XDPModeLinkDriver
 	XDPModeNative = "native"
 
+	// XDPModeBestEffort for loading progs with XDPModeLinkDriver
+	XDPModeBestEffort = "best-effort"
```

Comment:
The additional value `best-effort` needs to be reflected in the Cilium CLI model too. It's used by the command `cilium status --verbose`.

https://github.com/cilium/cilium/blob/main/api/v1/models/kube_proxy_replacement.go#L1137-L1138
&
https://github.com/cilium/cilium/blob/main/api/v1/models/kube_proxy_replacement.go#L1195-L1205

Reply:
No matter what I did I could not figure out how to get those files to be updated. Here's what I ran:

```
make generate-api
```

However, having these updated does not appear to be necessary. Here's the output from `cilium status --verbose`:

```
KubeProxyReplacement Details:
  Status:                 Strict
  Socket LB:              Enabled
  Socket LB Tracing:      Enabled
  Socket LB Coverage:     Full
  Devices:                ens785np0 10.121.160.29 (Direct Routing)
  Mode:                   SNAT
  Backend Selection:      Random
  Session Affinity:       Enabled
  Graceful Termination:   Enabled
  NAT46/64 Support:       Disabled
  XDP Acceleration:       Best-Effort
  Services:
  - ClusterIP:      Enabled
  - NodePort:       Enabled (Range: 30000-32767)
  - LoadBalancer:   Enabled
  - externalIPs:    Enabled
  - HostPort:       Enabled
```

Reply:
You should be able to update the [OpenAPI model](https://github.com/cilium/cilium/blob/main/api/v1/openapi.yaml#L2335) and re-generate with `make generate-api`.

At least for the sake of completeness

Reply:
That is very helpful, thank you @mhofstetter